HintEditor/HintEditor/CMakeLists.txt

```diff
@@ -0,0 +1,91 @@
+cmake_minimum_required(VERSION 3.3)
+project(HintEditor Swift)
+
+include(BundleUtilities)
+
+set(CMAKE_Swift_LANGUAGE_VERSION 3.0.1)
+
```

Comment:
Lets add this here:
```
# use latest macOS SDK
set(CMAKE_OSX_SYSROOT macosx)

# set minimum deployment target
set(CMAKE_OSX_DEPLOYMENT_TARGET "10.10")
```
To make sure we are building using latest macOS SDK and not lower than 10.10